src/app/credExplorer/basicPagerank.js

```diff
@@ -113,7 +113,20 @@ export function graphToOrderedSparseMarkovChain(
   );
 }
 
-function findStationaryDistribution(chain: SparseMarkovChain): Distribution {
+function findStationaryDistribution(
+  chain: SparseMarkovChain,
+  options?: {|
+    +verbose?: boolean,
+    +convergenceThreshold?: number,
+    +maxIterations?: number,
+  |}
+): Distribution {
+  const fullOptions = {
+    verbose: false,
+    convergenceThreshold: 1e-7,
+    maxIterations: 255,
+    ...(options || {}),
```

Comment:
Nice pattern. I keep being impressed with how much can be accomplished via spreads

Reply:
:+1: